# PACC 使能
int8 enable

# PACC 功能有效
int8 valid

float32 slope_dec

float32 slope_smooth

float32 slope_length

float32 distance_to_slope

float32 cc_upper_limit_v

pacc_chassis_input chassis_input
pacc_map_input map_input
pacc_internal internal
pacc_output output

float32 tar_throttle_ufilter

float32 cur_slope_ufilter

float32 engine_speed_ufilter